engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFBO.java

```diff
@@ -114,20 +112,13 @@ public StateChange getDefaultInstance() {
     }
 
     private int fetchTextureId() {
-
         // TODO: make checks to verify the FBOs has the requested buffer;
-
         if (inputFbo != null) {
-
             switch (textureType) {
-
                 case ColorTexture:
                     return inputFbo.colorBufferTextureId;
 
-                case DepthRenderBuffer:
-                    return inputFbo.depthStencilRboId;
-
-                case DepthTexture:
+                case DepthStencilTexture:
```

Comment:
Understood, so it was actually a bug to keep it there as it wouldn't have worked with glBindTexture.

Can you put a comment about it? Something like:

```java
// depthStencilRboId is a possible FBO attachment but is not covered by a case here 
// as it wouldn't work with the glBindTexture(TEXTURE_2D) call.
```